api/routes/scripts.py

```diff
@@ -1,10 +1,28 @@
 from flask import jsonify, request
-
+from flask import request
+import hashlib
+import json
 
 def get_error_response(e, status_code):
     return jsonify(e), status_code
 
+def get_good_response(result):
```

Comment:
Won't flask itself return a proper success status code if we just do "return results"?